libpromises/policyserver.c

```diff
@@ -0,0 +1,300 @@
+/*
+   Copyright (C) CFEngine AS
+
+   This file is part of CFEngine 3 - written and maintained by CFEngine AS.
+
+   This program is free software; you can redistribute it and/or modify it
+   under the terms of the GNU General Public License as published by the
+   Free Software Foundation; version 3.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+  To the extent this program is licensed as part of the Enterprise
+  versions of CFEngine, the applicable Commercial Open Source License
+  (COSL) may apply to this file if you as a licensee so wish it. See
+  included file COSL.txt.
+*/
+
+#include <policyserver.h>
+
+#include <eval_context.h>
+#include <addr_lib.h>
+#include <communication.h>
+
+#include <assert.h>
+#include <logging.h>
+#include <string_lib.h>
+#include <file_lib.h>
+
+//*******************************************************************
+// POLICY SERVER VARIABLES:
+//*******************************************************************
+
+static char *POLICY_SERVER      = NULL;
```

Comment:
Add a comment that `POLICY_SERVER` is the exact string to the one we bootstrapped to and can be IP or hostname and includes port. And another comment that `POLICY_SERVER_HOST` has the port removed, and can be host or IP. If I undestand this right.